docs/docs/guide/plugin-management.md

```diff
@@ -553,10 +553,10 @@ you can [add the new variant as a separate plugin](#multiple-variants) or switch
 1. Reinstall the plugin from the new `pip_url` using [`meltano install`](/reference/command-line-interface#install):
 
    ```bash
-   meltano install <type> <name>
+   meltano install [<type>|-] <name>
 
    # For example:
-   meltano install loader target-postgres
+   meltano install target-postgres
```

Comment:
Oops - will fix this.

Reply:
https://github.com/meltano/meltano/pull/8355/commits/9af5e19bf19d2794c30a776be01af7faeb8272fa